src/QA/QA.Web/Views/Review/ReadonlyReview.cshtml

```diff
@@ -166,9 +166,14 @@
                 <h3 class="govuk-heading-s">Description</h3>
                 <p class="govuk-body">@Model.EmployerDescription</p>
             </div>
-            <div>
+            <div id="@Anchors.EmployerName">
                 <h3 class="govuk-heading-s">Employer</h3>
                 <p class="govuk-body">@Model.EmployerName</p>
+                
```

Comment:
Why are we getting EmployerName from 2 different places? @Model.EmployerName and @Anchors.EmployerName

Reply:
Model.EmployerName is the name of the employer
Anchors.EmployerName is the name of the anchor